Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -32,14 +28,19 @@ final class PledgePaymentMethodsViewController: UIViewController {
   }
 
   private func configureSubviews() {
-    _ = (self.rootStackView, self.view)
+    _ = (self.cardsStackView, self.scrollView)
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToEdgesInParent()
 
-    _ = ([self.applePayButton, self.titleLabel, self.collectionView], self.rootStackView)
+    _ = ([self.applePayButton, self.titleLabel, self.scrollView], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
+    _ = (self.rootStackView, self.view)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
     NSLayoutConstraint.activate([
```

Comment:
Can we put all the constraints in a separate function `func setupConstraints()`?